File: Easy/Python/2485. Find the Pivot Integer.py
class Solution(object):
    def pivotInteger(self, n):
        pivot = -1
        left = 0

        for i in range(n + 1):
            left = left + i
            right = 0

            for j in range(i, n + 1):
                right = right + j
            
            if left == right:
                pivot = i
        
        return pivot

File: Easy/Java/2485. Find the Pivot Integer.java
class Solution {
    public int pivotInteger(int n) {
        int pivot = -1;
        int left = 0;

        for(int i = 0; i <= n; i++){
            left += i;
            int right = 0;
            
            for(int j = i; j <= n; j++)
                right += j;
            
            if(left == right)
                pivot = i;
        }

        return pivot;
    }
}
